src/components/Navigator/NavigatorCard.vue

```diff
@@ -1054,7 +1054,7 @@ $navigator-head-background-active: var(--color-fill-tertiary) !default;
   }
 
   .navigator-card-inner {
-    height: calc(100vh - #{$nav-height} - #{$filter-height});
```

Comment:
The bug was caused by this line accidentally taking precedence over the 100% height rule for the mobile style defined in the CardInner component, right?

(Just making sure I understand what went wrong and how you fixed it)

Reply:
yes